packages/flutter/test/material/buttons_test.dart

```diff
@@ -44,6 +44,46 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('Does FlatButton scale with font scale changes', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new Directionality(
+        textDirection: TextDirection.ltr,
+        child: new Material(
+          child: new MediaQuery(
+            data: new MediaQueryData(textScaleFactor: 1.0),
+            child: new Center(
+              child: new FlatButton(
+                  onPressed: () {},
```

Comment:
the most trivial nit ever: we use `{ }` for empty code blocks and `{}` for empty maps

Reply:
Oooh, tricky.  Done.